AppController/djinn.rb

```diff
@@ -2066,6 +2071,7 @@ def job_start(secret)
     # From here on we have the basic local state that allows to operate.
     # In particular we know our roles, and the deployment layout. Let's
     # start attaching any permanent disk we may have associated with us.
+    start_infrastructure_manager
```

Comment:
partially: i wanted to get more distance to see the error. And it is not needed till now, so there is no reason to start it before we get more info.